tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
+                 "dockerskeleton" \
+                 "nodejs6action"  \
+                 "python2action"  \
+                 "python3action"  \
+                 "swift3action"   \
+                 "java8action"
+do
+  docker tag "${docker_image_prefix}/${container}" "${dockerhub_image_prefix}/${container}:latest"
+  docker tag "${docker_image_prefix}/${container}" "${dockerhub_image_prefix}/${container}:${TRAVIS_BRANCH}-${TRAVIS_COMMIT::7}"
+  docker push "${dockerhub_image_prefix}/${container}"
+done
```

Comment:
Hi @markusthoemmes I tried using gradlew, but the command is pushing images I don't want to push to docker hub
```
dockerhub_image_prefix=openwhisk
./gradlew distDocker -PdockerImagePrefix=${dockerhub_image_prefix} -PdockerRegistry=docker.io
```
```
👾  $ cat ~/Desktop/log.txt | grep "docker push"
Executing 'docker push docker.io/csantanapr/scala:latest'
Executing 'docker push docker.io/csantanapr/dockerskeleton:latest'
Executing 'docker push docker.io/csantanapr/controller:latest'
Executing 'docker push docker.io/csantanapr/invoker:latest'
Executing 'docker push docker.io/csantanapr/java8action:latest'
Executing 'docker push docker.io/csantanapr/nodejs6action:latest'
Executing 'docker push docker.io/csantanapr/nodejsactionbase:latest'
Executing 'docker push docker.io/csantanapr/python2action:latest'
Executing 'docker push docker.io/csantanapr/python3action:latest'
Executing 'docker push docker.io/csantanapr/swift3action:latest'
Executing 'docker push docker.io/csantanapr/example:latest'
Executing 'docker push docker.io/csantanapr/cli:latest'
Executing 'docker push docker.io/csantanapr/badaction:latest'
Executing 'docker push docker.io/csantanapr/badproxy:latest'
```

How do I exclude some images? like `scala:test`, `nodejsactionbase:latest` , `cli:latest`, `adaction:latest`, `badaction:latest`